tests/TestCase/Database/QueryCompilerTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * cakephp(tm) : rapid development framework (https://cakephp.org)
+ * copyright (c) cake software foundation, inc. (https://cakefoundation.org)
+ *
+ * licensed under the mit license
+ * for full copyright and license information, please see the license.txt
+ * redistributions of files must retain the above copyright notice.
```

Comment:
Copyright block shouldn't be all lowercase

Reply:
I copied that from the QueryTest file so it needs to be adjusted there as well. Will do that now